pkg/harvester/edit/harvesterhci.io.host/index.vue

```diff
@@ -505,7 +505,9 @@ export default {
         }
       };
 
-      await retrySave();
+      if (this.longhornNode) {
+        await retrySave();
```

Comment:
Just curious, could we skip the whole func? (means Do call `saveLonghornNode()` if this node is witness node)

Reply:
Put `longhornNode` computed property in created, if user directly access https://localhost:8005/harvester/c/local/harvesterhci.io.host/harvester-node-0?mode=edit#labels, the longhornNode got undefined and miss the registration

<img width="1491" alt="Screenshot 2024-07-02 at 3 30 51 PM" src="https://github.com/harvester/dashboard/assets/5744158/fa7e3c75-e3d2-4d7b-93f1-343e02ba927c">

``` 
created(){ 
 ....
 if (this.registerAfterHook) {
      this.registerAfterHook(this.saveDisk);
      if (this.longhornNode) {
        this.registerAfterHook(this.saveLonghornNode);
      }
    }
}
```